ee-feature-pack/layer-metadata-tests/pom.xml

```diff
@@ -26,6 +26,12 @@
     <description>Tests for the layer metadata rules</description>
     <packaging>jar</packaging>
 
+    <properties>
+        <!-- Don't try to deploy the module -->
+        <maven.deploy.skip>true</maven.deploy.skip>
```

Comment:
Is there any reason to keep this property around, my understanding is that this is no longer consumed anywhere?

Reply:
Gotcha, I should have been more specific, i.e.  "will no longer be consumed once we upgrade to jboss-parent 50 since it disables that default execution".

Nevertheless, I am still not personally convinced, I will just remove the unused property from the projects I manage to avoid confusion.